articles/getting-started-with-browser-sync/index.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: engineering-education
+status: review
+published: false
+url: /engineering-education/getting-started-with-browser-sync/
+title: Getting Started with Browser-sync
+description: The basic concept of browser-sync, installing browser-sync and using it serve your files while building a website project
+author: aransiola-ayodele-leom
+date: 2020-09-24T00:00:27-21
+topics: [Browser-sync]
+excerpt_separator: <!--more-->
+images:
+ 
+ - url: /engineering-education/getting-started-with-browser-sync/hero.jpg
+   alt: Computers in a workspace displaying code
+---
+ 
+# Getting Started with Browser-sync
+ 
+ 
+Browser-sync is an automation tool that allows you to keep multiple browsers & devices in sync when building websites. It also gives your testing workflow faster synchronization, interaction, and code change across multiple devices easily, both internally and externally.
+ 
+>I have been working with different local servers just to replace the hassle of the live server. After trying out many, I found [browser-sync](https://www.browsersync.io/) more fascinating because of its complete packages. A stitch in time saves nine, which is Browser-sync in the quote.
```

Comment:
You've still not changed server to server**s**. Remove the before it.

To make the most sense: "I've been trying out different local servers just to replace the hassle of using live servers. I found that Browser-sync was the best because it has so many useful features. A stitch in time saves nine which really sums it up.